bin/alluxio-start.sh

```diff
@@ -309,6 +322,7 @@ main() {
         stop local
         sleep 1
       fi
+      start_logserver
```

Comment:
should we also include a way to start just the log server?

Reply:
Done.